packages/vscode-extension/src/devices/AndroidEmulatorDevice.ts

```diff
@@ -106,10 +114,20 @@ export class AndroidEmulatorDevice extends DeviceBase {
     }
   }
 
-  private async ensureOldEmulatorProcessExited() {
+  public static async ensureOldEmulatorProcessExited(avdId: string) {
     let runningPid: string | undefined;
-    const subprocess = exec("ps", ["-Ao", "pid,command"]);
-    const regexpPattern = new RegExp(`(\\d+)\\s.*qemu.*-avd ${this.avdId}`);
+    const command = Platform.select({
+      macos: "ps",
+      windows:
+        'powershell.exe "Get-WmiObject Win32_Process | Select-Object ProcessId, CommandLine | ConvertTo-Csv -NoTypeInformation"',
+    });
+    const args = Platform.select({ macos: ["-Ao", "pid,command"], windows: [] });
+    const subprocess = exec(command, args);
+    const pattern = Platform.select({
+      macos: `(\\d+)\\s.*qemu.*-avd ${avdId}`,
+      windows: `"(\\d+)".*qemu.*-avd ${avdId}`,
```

Comment:
Why pattern is different between platforms? Shouldn't `.*` catch `\s` as well?

Reply:
Yes, a pattern without \s is equivalent to the current.